src/types/queries.rs

```diff
@@ -242,26 +243,46 @@ pub struct PendingUpdate {
     pub effect:         PendingUpdateEffect,
 }
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, SerdeSerialize, SerdeDeserialize)]
+#[serde(tag = "updateType", content = "update")]
 pub enum PendingUpdateEffect {
+    #[serde(rename = "root")]
     RootKeys(HigherLevelAccessStructure<RootKeysKind>),
+    #[serde(rename = "level1")]
     Level1Keys(HigherLevelAccessStructure<Level1KeysKind>),
+    #[serde(rename = "level2V0")]
     Level2KeysCPV0(Authorizations<ChainParameterVersion0>),
+    #[serde(rename = "level2V1")]
     Level2KeysCPV1(Authorizations<ChainParameterVersion1>),
+    #[serde(rename = "protocol")]
     Protocol(ProtocolUpdate),
+    #[serde(rename = "electionDifficulty")]
     ElectionDifficulty(ElectionDifficulty),
+    #[serde(rename = "euroPerEnergy")]
     EuroPerEnergy(ExchangeRate),
+    #[serde(rename = "microGTUPerEuro")]
```

Comment:
Since this have other changes from the JSON serialization in gRPC v1, we could also fix the mentioning of GTU
```suggestion
    #[serde(rename = "microCcdPerEuro")]
```

Reply:
Good point.